implementation/test/TBTCSystemTest.js

```diff
@@ -24,24 +29,28 @@ describe("TBTCSystem", async function() {
     // Refer to this correctly throughout the rest of the test.
     tbtcSystem = tbtcSystemStub
     ecdsaKeepFactory = ecdsaKeepFactoryStub
+    tdt = tbtcDepositToken
   })
 
   describe("requestNewKeep()", async () => {
     let openKeepFee
+    const keepOwner = accounts[2]
+    const tdtOwner = accounts[1]
```

Comment:
I'm sorry, the order `2`, `1` makes me anxious. Can we switch these?
```js
const tdtOwner = accounts[1]
const keepOwner = accounts[2] 
```

Reply:
done